src/SFA.DAS.Roatp.CourseManagement.Web/Models/ProviderContact/ProviderContactSessionModel.cs

```diff
@@ -11,4 +11,6 @@ public class ProviderContactSessionModel
 
     public bool? UpdateExistingStandards { get; set; }
 
+    public bool? HasNoStandards { get; set; }
```

Comment:
Does this need to be nullable? This could be determined when we are initialising this model.